api/server.js

```diff
@@ -2,6 +2,9 @@ const environment = require('./src/environment');
 const serverChecks = require('@medic/server-checks');
 const logger = require('./src/logger');
 const express = require('express');
+const morgan = require('morgan');
+const moment = require('moment');
+morgan.token('date', () => moment().format('YYYY-MM-DD HH:mm:ss'));
```

Comment:
Is this enough precision? Surely millis would be useful?

Reply:
I was reluctant to change the format, in case _someone_ is already watching dates from winston logs. 
I'm open to adding millis for sure!

Reply:
It's probably not too important, but I slightly prefer millis, and it only costs us a few bytes...

Reply:
I agree, I'll change that and add the e2e test today.